### PR TITLE
Added homeworks 1, 2 and 3

### PR DESCRIPTION
**Pull request by amiller-gh**

### CODE REVIEW

File: homework.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (amiller-gh) [inline homework.html:1]: @lemonhawk21 When you open this file in the browser, you'll be able to see the `console.log()` output of your homework!

File: homework/hw0.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+startHomework();
```
Comment (amiller-gh) [inline homework/hw0.js:1]: Ignore the scaffolding code (ex: `startHomework()`, `problem('name', ... )` ) in these files, all you have to care about is are comments with problem directions and the `// <Code Goes Here>` sections where you work through the answers! When you look at `homework.html` in your browser, you'll see your code run with pretty comments telling you where each homework / problem starts and ends 😃  We can chat about the quick scaffolding code I wrote later if you'd like.

File: util/hw-utils.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// You don't need to touch this file ^_^
```
Comment (amiller-gh) [inline util/hw-utils.js:1]: This is the REEEALLY quick scaffolding code. You can ignore this for now 😃

File: homework/hw1.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  //   - notBad('This dinner is bad!'): 'This dinner is bad!'
+  /******************************************************************************/
+
+  // <Code Goes Here>
+
+
+});
+
+
+problem('notBad', () => {
```
Comment (melissaroman) [inline homework/hw1.js:79]: This should be called `fixStart`

**Pull request merged**